TenantFile/Api/.config/dotnet-tools.json

```
@@ -0,0 +1,12 @@
+{
+  "version": 1,
+  "isRoot": true,
+  "tools": {
+    "dotnet-ef": {
+      "version": "5.0.0-rc.1.20451.13",
```

This is out of preview now 
```suggestion
      "version": "5.0.0",
```

---

Yeah that's my bad. I was waiting for features to drop from one package so I could implement another thing on a different package and it all just got entangled in this branch. I'll work on splitting it up.